src/examples.rs

```diff
@@ -194,3 +193,35 @@ pub fn great_boy_pod_full_flow() -> MainPodBuilder {
         alice,
     )
 }
+
+// Tickets
+
+pub fn tickets_sign_pod_builder(params: &Params) -> SignedPodBuilder {
+    // Create a signed pod with all atomic types (string, int, bool)
+    let mut builder = SignedPodBuilder::new(params);
+    builder.insert("eventId", 123);
+    builder.insert("productId", 456);
+    builder.insert("attendeeName", "John Doe");
+    builder.insert("attendeeEmail", "john.doe@example.com");
+    builder.insert("isConsumed", true);
+    builder.insert("isRevoked", false);
```

Comment:
question from my ignorance on the POD system:
how is the POD revoked? since the `isRevoked` field is part of the key-values of the SignedPOD, in that case, the POD.id will change so it's no longer the original POD but a new one, right?
(same applies for the field `isConsumed`)

Reply:
good question -- this is a bit of a Zupass-specific example, but we have a protocol called 'feeds' that continually polls PODs from a server every minute, and the server re-issues certain PODs to a given user (there is an external stable "PCD ID" to map original PODs -> new PODs)